storage/tianmu/core/engine.cpp

```diff
@@ -194,6 +194,9 @@ Engine::Engine()
                        tianmu_sysvar_load_threads ? tianmu_sysvar_load_threads : std::thread::hardware_concurrency()),
       query_thread_pool(
           "query", tianmu_sysvar_query_threads ? tianmu_sysvar_query_threads : std::thread::hardware_concurrency()),
+      delete_or_update_thread_pool("delete_or_update", tianmu_sysvar_delete_or_update_threads
```

Comment:
Here, you should make sure the pool size is always a postive number. and what we will do if the `tianmu_sysvar_delete_or_update_threads` is set to a negtive number. Because we do no thave the check function avaiable here.